self-hosted-login/okta-aspnetcore-mvc-example/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@using Microsoft.Extensions.Options;
+@inject IOptions<OktaSettings> oktaSettings;
```

Comment:
If you do this, do you need to also inject it at the controller level in AccountController?

Any benefit of doing it this way instead of creating a view model and doing:

```
return new View(new LoginViewModel
{
  OktaDomain = oktaSettings.Value.OktaDomain
}
```

That might be a little cleaner.